sdk/typescript/src/cryptography/index.ts

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+export * from './signature.js';
+export * from './mnemonics.js';
+
+export { PublicKey } from './publickey.js';
+export { BaseSigner as Signer, Keypair, ExportedKeypair } from './keypair.js';
```

Comment:
there was already a `Signer` being exported from the root index.ts file, so adding another Signer caused issues.  Once that signer is removed, this abstract class should be renamed to Signer